pkg/pillar/types/downloadertypes.go

```diff
@@ -25,7 +26,7 @@ type DownloaderConfig struct {
 }
 
 func (config DownloaderConfig) Key() string {
-	return config.ImageID.String()
+	return config.ImageSha256
```

Comment:
I suspect the comments for DownloaderConfig and DownloaderStatus should be updated because they specify the key
Also, we tend to put the Key first int the struct.

Also, is the imageID still used for something in the downloader? If it is only for logging we can add a comment in the struct that it just for logging.

Reply:
I will update the comments.